### DISCUSSION TITLE
Can't use Panopticon to Update Joomla 4.4.3>4.4.4

### DISCUSSION THREAD

**Discussion by altairtg** [akeeba/panopticon#633] [Q&A (Community Support)]
I try to use Panopticon 1.1.3 to make update, but I'm, stuck...
After full Backup I recive error:
**Invalid JSON response from the update extraction script; the site may be broken.**

In context information is:
{
    "file": "C:\\xampp-new\\htdocs\\panopticon2024\\src\\Task\\JoomlaUpdate.php",
    "line": 1354,
    "trace": "#0 C:\\xampp-new\\htdocs\\panopticon2024\\src\\Task\\JoomlaUpdate.php(1257): Akeeba\\Panopticon\\Task\\JoomlaUpdate->doExtractAjax(Object(Akeeba\\Panopticon\\Model\\Site), Object(Awf\\Registry\\Registry), Array)\n#1 C:\\xampp-new\\htdocs\\panopticon2024\\src\\Task\\JoomlaUpdate.php(965): Akeeba\\Panopticon\\Task\\JoomlaUpdate->j404ExtractStart(Object(Akeeba\\Panopticon\\Model\\Site), Object(Awf\\Registry\\Registry))\n#2 C:\\xampp-new\\htdocs\\panopticon2024\\src\\Task\\JoomlaUpdate.php(81): Akeeba\\Panopticon\\Task\\JoomlaUpdate->runExtract(Object(stdClass), Object(Awf\\Registry\\Registry))\n#3 C:\\xampp-new\\htdocs\\panopticon2024\\src\\Model\\Task.php(457): Akeeba\\Panopticon\\Task\\JoomlaUpdate->__invoke(Object(stdClass), Object(Awf\\Registry\\Registry))\n#4 C:\\xampp-new\\htdocs\\panopticon2024\\src\\CliCommand\\TaskRun.php(90): Akeeba\\Panopticon\\Model\\Task->runNextTask(Object(Akeeba\\Panopticon\\Library\\Logger\\ForkedLogger), Object(Symfony\\Component\\Console\\Style\\SymfonyStyle))\n#5 C:\\xampp-new\\htdocs\\panopticon2024\\vendor\\symfony\\console\\Command\\Command.php(326): Akeeba\\Panopticon\\CliCommand\\TaskRun->execute(Object(Symfony\\Component\\Console\\Input\\ArgvInput), Object(Symfony\\Component\\Console\\Output\\ConsoleOutput))\n#6 C:\\xampp-new\\htdocs\\panopticon2024\\vendor\\symfony\\console\\Application.php(1078): Symfony\\Component\\Console\\Command\\Command->run(Object(Symfony\\Component\\Console\\Input\\ArgvInput), Object(Symfony\\Component\\Console\\Output\\ConsoleOutput))\n#7 C:\\xampp-new\\htdocs\\panopticon2024\\vendor\\symfony\\console\\Application.php(324): Symfony\\Component\\Console\\Application->doRunCommand(Object(Akeeba\\Panopticon\\CliCommand\\TaskRun), Object(Symfony\\Component\\Console\\Input\\ArgvInput), Object(Symfony\\Component\\Console\\Output\\ConsoleOutput))\n#8 C:\\xampp-new\\htdocs\\panopticon2024\\vendor\\symfony\\console\\Application.php(175): Symfony\\Component\\Console\\Application->doRun(Object(Symfony\\Component\\Console\\Input\\ArgvInput), Object(Symfony\\Component\\Console\\Output\\ConsoleOutput))\n#9 C:\\xampp-new\\htdocs\\panopticon2024\\cli\\panopticon.php(82): Symfony\\Component\\Console\\Application->run()\n#10 C:\\xampp-new\\htdocs\\panopticon2024\\cli\\panopticon.php(83): {closure}()\n#11 {main}"
}

How to resolve this?

**Comment by nikosdion** [comment]
Schedule the update again. If it doesn't work, there is something broken in Joomla itself. You can try running the update directly at the site and see if it keeps breaking, in which case you need to check with your host.

**Comment by altairtg** [comment]
Hi Nikosdion,
I try on 3 sites, everytime I recive: 
Failed to update the core CMS
#0. Invalid JSON response from the update extraction script; the site may be broken. 

Extensions  I can update without problems.
When I try update Joomla on one site via backend all was OK without any problems.

**Reply by nikosdion** [reply]
Which exact Joomla! version are we talking about?

If they were Joomla! 4.4.3 or 5.0.3 sites then the problem is definitely on the hosting side. The requests made by Panopticon are identical to those made by Joomla! itself; remember, I wrote that part of Joomla! Update and know full well how it works. If the same request isn't accepted from one origin but accepted from another the only conclusion I can reach is that there is some kind of server configuration at fault.

**Comment by altairtg** [comment]
Hi!
Which version Joomla! see topic :) I wrote: Joomla 4.4.3>4.4.4

Regarding your sugestion to check server configuraton: I have about 70 sites to update on many servers on different hostings. In all servers are something wrong? Maybe, I don't know. It was faster to update through the backend than to look for the problem on the servers.
Maybe problem was quite big update pack (28MB)? Maybe problem was SecurityCheck...
I must check how update will be performed on clean Joomla installation.

**Comment by nikosdion** [comment]
I wanted to make sure all sites are the same version. You mentioned a third party security solution. It is possible that it restricts access to Joomla's extract.php which handles the extraction of the update only to IPs which have visited the administrator/index.php recently. Of course and as I said it is better to ask your host as neither you who are hosted there, nor me who doesn't even know which host we are talking about, know how your host has configured their servers. All I can tell you is that I've updated two dozen sites which are hosted on Rochen, CloudAccess, and servers I have set up myself. Zero problems.Nicholas K. DionysopoulosLead developer and directorAkeeba LtdSent from a mobile device. Please excuse typos and brevity.

**Comment by altairtg** [comment]
All sites was in the same version 4.4.3. I supose that problem is third party security solution - Securitycheck. It's like your Admin tools.
The developer is fine and responds quickly, so if I learn something, I will of course share this knowledge.
